pkg/scheduler/apis/config/types.go

```diff
@@ -55,6 +56,16 @@ type KubeSchedulerConfiguration struct {
 	// HardPodAffinitySymmetricWeight represents the weight of implicit PreferredDuringScheduling affinity rule, in the range 0-100.
 	HardPodAffinitySymmetricWeight int32
 
+	// TopologySpreadConstraints is the cluster-level configuration for
+	// topology-based pod spreading.
+	// Constraints are applied to pods that don't define any in their PodSpec.
+	// Label selectors must be empty, as they are deduced from the pods'
+	// membership in Services, Replication Controllers, Replica sets or Stateful
+	// sets.
+	// Omit to use the default: soft spreading among nodes and zones.
+	// Set to empty to disable.
```

Comment:
> Set to empty to disable.

that's no longer accurate if defaulting is on len(...) == 0, right?

Reply:
Updated